Infrastructure/Services/ComplaintLogService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Core.Entities;
+using Core.Interfaces;
+using Infrastructure.Services.BasicCrudServices;
+using System;
+
+namespace Infrastructure.Services
+{
+    public class ComplaintLogService : CreateService<ComplaintLog>, IComplaintLogService
+    {
+        public ComplaintLogService(Donos_Context context) : base(context)
+        {
+        }
+
+        public void CancelComplaint(Guid complaintId)
+        {
+            var complaintlog = this.context.ComplaintsLogs.Find(complaintId);
+            if(complaintlog != null)
```

Comment:
Brakująca spacja między idem a nawiasem

Reply:
idk czy to jest dostępne w .net core 3.1 (ale raczej tak), ale istnieje `is not` pattern - można rozważyć jego użycie (chyba jest on jakoś lepiej zoptymalizowany niż zwykłe porównanie)